src/core/Akka.Persistence.Tests/SnapshotFailureRobustnessSpec.cs

```diff
@@ -161,11 +162,7 @@ protected override void Save(SnapshotMetadata metadata, object payload)
                 {
                     var bytes = Encoding.UTF8.GetBytes("b0rk");
                     var tempFile = WithOutputStream(metadata, stream => stream.Write(bytes, 0, bytes.Length));
-                    var newTempFileName = GetSnapshotFileForWrite(metadata, ".tmp");
-                    if (tempFile != newTempFileName)
-                    {
-                        tempFile.MoveTo(newTempFileName.FullName);
-                    }
+					tempFile.MoveTo(GetSnapshotFileForWrite(metadata, "").FullName);
```

Comment:
We should not use `.tmp` extension here, because the original file (with the same name) has it